### DISCUSSION TITLE
Custom text on the OQRS page

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#1947] [Ideas / Feature Requests]
It would be good to have custom text on the OQRS page. For example to describe OQRS policy or, in my case, how to select correct station location in addition to policy.

**Comment by AndreasK79** [comment]
Edit station location and go to the bottom. I have included a text column there for custom text.

**Reply by R1BLH** [reply]
@AndreasK79 
> Edit station location and go to the bottom. I have included a text column there for custom text.

This text is shown only on the stage of the finalizing of the QSL request. 

I mean common text for all station locations.
For example: common QSL request policy and (in my case) how to find correct station location:
[date of finish]-[date of start]-[QTH locator]-[Callsign]]: 230131-230121-KP50da-R1BLH

![2023-01-23_20-33-13](https://user-images.githubusercontent.com/16667173/214109358-385209d3-ed39-4cd2-af5e-fcbece0e6d68.png)

**Reply by AndreasK79** [reply]
I understand. I never implemented that. Will add that to my list.

**Reply by R1BLH** [reply]
> I understand. I never implemented that. Will add that to my list.

Thank you!

**Reply by AndreasK79** [reply]
@cats-shadow custom text is added now. You can find it under admin -> global options -> oqrs.

**Reply by R1BLH** [reply]
Thank you!

**Reply by R1BLH** [reply]
Could you advise how to make multiline text? (add a CR LF codes to the line?)

**Reply by AndreasK79** [reply]
Html tags should work. Try `<br>`

**Reply by R1BLH** [reply]
HTML tags work! Thanks!
